wpilibc/src/main/native/cpp/Filesystem.cpp

```diff
@@ -27,5 +27,11 @@ void frc::filesystem::GetOperatingDirectory(
 
 void frc::filesystem::GetDeployDirectory(wpi::SmallVectorImpl<char>& result) {
   frc::filesystem::GetOperatingDirectory(result);
-  wpi::sys::path::append(result, "deploy");
+  if (RobotBase::IsReal()) {
```

Comment:
if constexpr?

Reply:
Changed, also also changed the other one in this file to use if constexpr as well.